src/alto-ui/Actions/__tests__/Actions.test.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Actions from '../Actions';
+import Dropdown from '../../Dropdown';
+import Group from '../../Group';
+
+jest.mock('../../Dropdown', () => {
```

Comment:
`jest.mock('../../Dropdown', () => ({ onClick }) => <button className="Dropdown" onClick={() => onClick()} />;`

Reply:
1, Missing tests are added
2. The Dropdown mock has been rebuilt